include/serialport.h

```diff
@@ -140,17 +140,20 @@ class CSerial {
 	bool InstallationSuccessful;// check after constructing. If
 								// something was wrong, delete it right away.
 
-	// Constructor takes com port number (0-3)
-	CSerial(Bitu id, CommandLine* cmd);
-	
+	/*
+	 * Communication port index is typically 0-3, but logically limited
+	 * to the number of physical interrupts available on the system.
+	 */
+	CSerial(const uint8_t port_index_, CommandLine *cmd);
```

Comment:
Same; I like your suggestion; will add that. 
(I added the underscore tails because they shadowed member variables.. I will try to adjust the other shadowed vars in the later effc++ commits too).

Reply:
Fixed in:
- commit d459365f fixup! Use uint32 instead of Bitu in softmodem class
- commit de41b594 fixup! Use uint32 instead of Bitu in serialport class
- commit 05cc0b57 fixup! Use uint32 instead of Bitu in serialdummy class
- commit 4365cbe7 fixup! Use uint32 instead of Bitu in nullmodem class
- commit e95e7d0d fixup! Use uint32 instead of Bitu in directserial class